cmd/openshift-install/create.go

```diff
@@ -323,6 +326,13 @@ func waitForInitializedCluster(ctx context.Context, config *rest.Config) error {
 		},
 	)
 
+	// If we timed out and the CVO failed, print out the first failing operator
```

Comment:
Nothing in the code below is `print out the first failing operator`. It is just printing the `Failing` condition of the `clusterversion` object...

Reply:
Fixed the comment :arrow_down: